nixos/modules/services/x11/xserver.nix

```diff
@@ -347,6 +347,7 @@ in
 
       xkbDir = mkOption {
         type = types.path;
+        default = "${pkgs.xkeyboard_config}/etc/X11/xkb";
```

Comment:
```services.xserver.xkbDir``` should be set irregardless of ```services.xserver.enable```
```plasma5``` and ```enlightment``` read ```services.xserver.xkbDir```

Reply:
^ should this be a code comment?

Reply:
I think, besides the commented reason to move, here is also the more "natural" place for the default value than under ```mkIf cfg.enable { }```. The latter would need a code comment.